test/gtest/conv_embed_db.cpp

```diff
@@ -0,0 +1,230 @@
+    TestCase{GetEnvVars(igemm_wrw), precision + flags + " --input 128 64 56 56 --weights 64 64 3 3 --pads_strides_dilations 1 1 1 1 1 1"},
+    TestCase{GetEnvVars(igemm_wrw), precision + flags + " --input 128 256 14 14 --weights 256 256 3 3 --pads_strides_dilations 1 1 1 1 1 1"},
+    TestCase{GetEnvVars(igemm_wrw), precision + flags + " --input 128 512 7 7   --weights 512 512 3 3 --pads_strides_dilations 1 1 1 1 1 1"},
+    TestCase{GetEnvVars(igemm_wrw), precision + flags + " --input 128 1024 14 14 --weights 512 1024 1 1 --pads_strides_dilations 0 0 2 2 1 1"},
+    TestCase{GetEnvVars(igemm_wrw), precision + flags + " --input 128 1024 14 14 --weights 2048 1024 1 1 --pads_strides_dilations 0 0 2 2 1 1"},
+    TestCase{GetEnvVars(igemm_fwd), precision + flags + " --input 128 256 14 14 --weights 1024 256 1 1 --pads_strides_dilations 0 0 1 1 1 1"},
+    TestCase{GetEnvVars(igemm_fwd), precision + flags + " --input 128 512 28 28 --weights 256 512 1 1 --pads_strides_dilations 0 0 2 2 1 1"},
+    TestCase{GetEnvVars(igemm_fwd), precision + flags + " --input 128 1024 14 14 --weights 256 1024 1 1 --pads_strides_dilations 0 0 1 1 1 1"},
+    TestCase{GetEnvVars(igemm_fwd_wrw), precision + flags + " --input 128 64 56 56 --weights 256 64 1 1 --pads_strides_dilations 0 0 1 1 1 1"},
+    TestCase{GetEnvVars(igemm_fwd_wrw), precision + flags + " --input 128 64 56 56 --weights 64 64 1 1 --pads_strides_dilations 0 0 1 1 1 1"},
+    TestCase{GetEnvVars(igemm_fwd_wrw), precision + flags + " --input 128 128 28 28 --weights 512 128 1 1 --pads_strides_dilations 0 0 1 1 1 1"},
+    TestCase{GetEnvVars(igemm_fwd_wrw), precision + flags + " --input 128 256 56 56 --weights 128 256 1 1 --pads_strides_dilations 0 0 2 2 1 1"},
+    TestCase{GetEnvVars(igemm_fwd_wrw), precision + flags + " --input 128 256 56 56 --weights 64 256 1 1 --pads_strides_dilations 0 0 1 1 1 1"},
+    TestCase{GetEnvVars(igemm_fwd_wrw), precision + flags + " --input 128 512 28 28 --weights 1024 512 1 1 --pads_strides_dilations 0 0 2 2 1 1"},
+    TestCase{GetEnvVars(igemm_fwd_wrw), precision + flags + " --input 128 512 28 28 --weights 128 512 1 1 --pads_strides_dilations 0 0 1 1 1 1"},
+    TestCase{GetEnvVars(igemm_fwd_wrw), precision + flags + " --input 128 512 7 7   --weights 2048 512 1 1 --pads_strides_dilations 0 0 1 1 1 1"},
+    TestCase{GetEnvVars(igemm_fwd_wrw), precision + flags + " --input 128 2048 7 7 --weights 512 2048 1 1 --pads_strides_dilations 0 0 1 1 1 1"}
+        // clang-format on
+    };
+
+    return test_cases;
+}
+
+INSTANTIATE_TEST_SUITE_P(Conv2dGroup, Conv2dFloat, testing::Values(GetTestCases("--float")));
+INSTANTIATE_TEST_SUITE_P(Conv2dGroup, Conv2dHalf, testing::Values(GetTestCases("--half")));
+INSTANTIATE_TEST_SUITE_P(Conv2dGroup, Conv2dInt8, testing::Values(GetTestCases("--int8")));
+INSTANTIATE_TEST_SUITE_P(Conv2dGroup, Conv2dBFloat16, testing::Values(GetTestCases("--bfloat16")));
```

Comment:
Test names should reflect that the test is intended for a library configuration with embedded databases.

Reply:
> Test names should reflect that the test is intended for a library configuration with embedded databases.

Test names have been updated. Could you resolve conversation if this has been resolved?

Reply:
[Resolved]